docs/gen-docs/kyma_alpha_create_module.md

```diff
@@ -42,13 +42,17 @@ Build module modB in version 3.2.1 and push it to a local registry "unsigned" su
       --channel string         Channel to use for the module template. (default "stable")
       --clean                  Remove the mod-path folder and all its contents at the end.
   -c, --credentials string     Basic authentication credentials for the given registry in the format user:password
+      --default-cr string      File containing the default Custom Resource of the module. If the module is a kubebuilder project, the default CR will be automatically detected.
```

Comment:
```suggestion
      --default-cr string      File containing the default custom resource of the module. If the module is a kubebuilder project, the default CR will be automatically detected.
```
See the [Terminology section of our guides](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology) for reference